package.json

```diff
@@ -132,13 +133,12 @@
 		"react-i18next": "^11.7.1"
 	},
 	"dependencies": {
-		"@fandom-frontend/design-system": "15.18.1",
-		"@fandom-frontend/react-common": "13.23.1",
+		"@fandom-frontend/design-system": "15.13.1",
+		"@fandom-frontend/react-common": "13.19.0",
```

Comment:
this is interesting because it looks like a downgrade 🤔

Reply:
Well, I've been updating from version 9 to 15 for example. A new version must have been added during my PTO

Reply:
It's fixed